Makefile

```diff
@@ -11,7 +11,11 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 
-.PHONY: all gobuild static docker release certs test clean netkitten test-registry run-functional-tests gremlin benchmark-test gogenerate run-integ-tests image-cleanup-test-images
+PAUSE_CONTAINER_IMAGE = "amazon/amazon-ecs-pause"
+PAUSE_CONTAINER_TAG = "0.1.0"
```

Comment:
As the path to the pause-container tarball is fixed, I was thinking along the lines of having a constant in our config. We could also add additional fields to our config struct to hold these  attributes:
* PauseContainerTarballPath
* PauseContainerTag 

However, I was thinking of following up with another PR for loading pause container on startup. This PR is just for packaging the pause-container tarball within the agent image.